src/isar/config/settings.env

```diff
@@ -14,7 +14,8 @@ ISAR_MQTT_SSL_ENABLED = true
 
 ISAR_AUTHENTICATION_ENABLED = false
 
-ISAR_ROBOT_ID = R2-D2
+ISAR_ROBOT_NAME = R2-D2
+ISAR_ISAR_ID = c68b679d-308b-460f-9fe0-87eaadbd8a6e
```

Comment:
These two does not need to be here as they are defaulted in `settings.py`

Reply:
done